share/completions/oksh.fish

```diff
@@ -0,0 +1,7 @@
+set -l options allexport notify noclobber errexit noglob trackall keyword monitor noexec privileged stdin nounset verbose markdirs xtrace
+complete -c oksh -a "+{a,b,C,e,f,h,i,k,l,m,n,p,r,u,v,X,x,o}"
+complete -c oksh -a "-{a,b,C,e,f,h,i,k,l,m,n,p,r,u,v,X,x,o}"
```

Comment:
These completions are really opaque to the point of not serving any purpose, and I would be against merging this as-is.

The completions you are specifying here are abbreviated versions of the options available for oksh's `set` builtin. I think they should be presented as short/long arguments with attached descriptions. (See [the man page](https://manned.org/oksh.1).)

It's up to you if you want to implement them or close out the PR.

Reply:
Fixed. :)